bitnami/common/templates/_secrets.tpl

```diff
@@ -55,3 +55,48 @@ Params:
 
 {{- printf "%s" $key -}}
 {{- end -}}
+
+{{/*
+Generate secret password or retrieve one if already created.
+
+Usage:
+{{ include "common.secrets.manage.password" (dict "secret" "secret-name" "key" "keyName" "customValues" (list .Values.path.to.the.password1 .Values.path.to.the.password2) "length" "password-length" "strong" false "context" $) }}
+
+Params:
+  - secret - String - Required - Name of the 'Secret' resource where the password is stored.
+  - key - String - Required - Name of the key in the secret.
+  - customValues - Optional - List of parameters that can be used by the user to define the password. Will pick first parameter with a defined value.
+  - length - int - Optional - Length of the generated random password
+  - strong - Boolean - Optional - Whether to add "_" symbols to the generated random password
```

Comment:
Why adding "_" would be strong?

Reply:
My bad, I didn't change this definition after my latests changes. It should say `Whether to add symbols...`

Reply:
Ok